application/src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -66,6 +66,8 @@ public class RoutingWorker {
   private PageCursorInput pageCursorInput = null;
 
   public RoutingWorker(OtpServerRequestContext serverContext, RouteRequest request, ZoneId zoneId) {
+    // Applying the page cursor modifies the request by removing the direct mode, for example.
+    // If the page cursor doesn't exist, this function does nothing.
```

Comment:
These comments should not be here. They belong in the JavaDoc of applyPageCursor. 

FYI: 
Avoid comment on code:
 - Does it belong in the JavaDoc?
 - Can I refactor to make my intension clear. Often introducing variables with a good name can replace a comment, if the logic is complex a "private" method with JavaDoc and UnitTest can replace a comment.
 - Learn your IDE, to show JavaDoc when needed (Ctrl + J will open JavaDoc of any element in IntelliJ).
 - Only if the above options is not good, should a inline code comment be used.

Reply:
I just removed this, the `applyPageCursorFactory` has sufficient comments and is clear enough